roles/cdi/templates/cdi-operator-cr.yaml.j2

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Use [this](https://github.com/kubevirt/containerized-data-importer/blob/master/manifests/templates/cdi-operator-cr.yaml.in) template instead

```
apiVersion: cdi.kubevirt.io/v1alpha1
kind: CDI
metadata:
  name: cdi
```

Reply:
done